src/frontend/packages/lib_common/src/queries/createOne.tsx

```diff
@@ -23,5 +27,5 @@ export const createOne: MutationFunction<
     throw { code: 'exception' };
   }
 
-  return await response.json();
+  return (await response.json()) as Promise<T>;
```

Comment:
```suggestion
  return (await response.json()) as T;
```

Reply:
Either that `return response.json() as Promise<T>;` or that `return (await response.json()) as T;`